Automattic/wp-calypso
# Plans: Move the wrapped components into individual components

Related to https://github.com/Automattic/wp-calypso/issues/87047

## Proposed Changes

This PR is the first step of addressing #87047. It cleans up the main index by moving the . Accompanying this change, the main `style.scss` has also been separated into a `_shared.scss`, `features-grid/style.scss` and `comparison-grid/style.scss`. The main reason is because of webpack module resolution would skip importing `style.scss` from my test after the file structure change. By restructuring the CSS files as well, we can make sure the relevant styles are imported correctly, with a side benefit that they now stay closer to their respective components.

## Why are these changes being made?


The end result of addressing https://github.com/Automattic/wp-calypso/issues/87047 is that the both wrapper components are deprecated. The 1st step introduced here is that the existence of the wrapper components are obscured from the main index.

## Testing Instructions



Run `yarn workspace @automattic/plans-grid-next storybook` and make sure both the features grid and the comparison grid render correctly.


## Pre-merge Checklist
